spec/graphql/analysis/max_query_complexity_spec.rb

```diff
@@ -50,7 +50,7 @@
     end
   end
 
-  describe "when complexity is overriden at query-level" do
```

Comment:
@esjee does it matter if you decrease or increase at query level? If not, you can use the original description.